Wikipedia/Code/StackedImageLabelView.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+/// A vertically stacked image/label group that resembles `UITableView`'s swipe actions 
+final class StackedImageLabelView: SetupView {
+
+    // MARK: - Properties
+
+    var increaseLabelTopPadding: Bool = false {
+        didSet {
+            labelTopAnchor.constant = increaseLabelTopPadding ? 8 : 2
+            setNeedsLayout()
+        }
+    }
+
+    private var imageDimension: CGFloat = 40
+    private var labelTopAnchor: NSLayoutConstraint = NSLayoutConstraint()
+
+    // MARK: - UI Elements
+
+    lazy var label: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: UIFont.systemFont(ofSize: 14, weight: .bold), maximumPointSize: 32)
```

Comment:
Hmm - it's a slightly odd to funnel through the `DynamicTextStyle` system because I don't believe there's a system `UIFont.TextStyle` that precisely matches the design characteristics we're looking for. But I can see the benefit of trying to centralize some of this scaled font behavior. I made a little method (`wmf_scaledSystemFont(...)`) on `UIFont` in that same source file to use if you think it's better, or if you had in mind something different/specific, please feel free to suggest.

Reply:
Yep, that looks good, I mainly just wanted to keep it to a single area to funnel it through in case we ever need to do a larger text change across the app.